electroncash/daemon.py

```diff
@@ -181,7 +180,7 @@ def init_server(self, config, fd, is_gui):
         server.register_function(self.ping, 'ping')
         server.register_function(self.run_gui, 'gui')
         server.register_function(self.run_daemon, 'daemon')
-        self.cmd_runner = Commands(self.config, None, self.network)
+        self.cmd_runner = Commands(self.config, None, self.network, self)
```

Comment:
It seems that a prerequisite for this PR is https://github.com/spesmilo/electrum/commit/de903103dafd46e1073d0e4de5df86e2407386bd.

i.e. **in electroncash_gui/qt/main_window.py**:

```diff
diff --git a/electroncash_gui/qt/main_window.py b/electroncash_gui/qt/main_window.py
index 8314162ba..93b3e20a1 100644
--- a/electroncash_gui/qt/main_window.py
+++ b/electroncash_gui/qt/main_window.py
@@ -3090,7 +3090,7 @@ class ElectrumWindow(QMainWindow, MessageBoxMixin, PrintError):
         console.updateNamespace({'util' : util, 'bitcoin':bitcoin})
 
         set_json = Weak(self.console.set_json)
-        c = commands.Commands(self.config, self.wallet, self.network, lambda: set_json(True))
+        c = commands.Commands(self.config, self.wallet, self.network, self.gui_object.daemon, lambda: set_json(True))
         methods = {}
         password_getter = Weak(self.password_dialog)
         def mkfunc(f, method):
```